app.py

```diff
@@ -6,64 +6,86 @@
 from flask_pymongo import PyMongo
 from bson.objectid import ObjectId
 import requests
+from datetime import datetime
 
 app = Flask(__name__)
 
 app.config['MONGO_DBNAME'] = 'root_directory'
 app.config['MONGO_URI'] = 'mongodb+srv://admin:OeMp96cSKTBrjtFz@cluster0-anyov.mongodb.net/root_directory?retryWrites=true&w=majority'
 mongo = PyMongo(app)
 
-@app.route('/api/v1/users/<int:user_id>', methods=['GET'])
+@app.route('/api/v1/users/<string:user_id>', methods=['GET'])
 def get_one_user(user_id):
     user = mongo.db.users.find_one_or_404({'_id': user_id})
     response = {
-        'user_id': user['_id'],
-        'user_name': user['name']
+        'id': user['_id'],
+        'userName': user['name'],
+        'timestamp': user['timestamp']
     }
-    return jsonify({'user': response})
+    return jsonify({response})
 
-@app.route('/api/v1/users/<int:user_id>/plants', methods=['GET'])
+@app.route('/api/v1/users/<string:user_id>/plants', methods=['GET'])
 def get_user_plants(user_id):
     user = mongo.db.users.find_one_or_404({'_id': user_id})
     user_plants = mongo.db.plants.find({'user_id': user_id})
     format_plant = lambda plant: {
-        'plant_id': plant['_id'],
-        'user_id': plant['user_id'],
-        'nickname': plant['nickname'],
-        'plant_type': plant['plant_type']
+        'id': plant['_id'],
+        'userId': plant['user_id'],
+        'plantName': plant['plant_name'],
+        'plantType': plant['plant_type'],
+        'imageURL': plant['image_url'],
+        'care': plant['care']
     }
     response = list(map(format_plant, list(user_plants)))
     return jsonify({'plants': response})
 
+@app.route('/api/v1/users/<string:user_id>/plants/<string:plant_id>', methods=['GET'])
+def get_one_plant(user_id, plant_id):
+    user = mongo.db.users.find_one_or_404({'_id': user_id})
+    plant = mongo.db.plants.find_one_or_404({'_id': plant_id})
+    response = {
+        'id': plant_id,
+        'userId': user_id,
+        'plantName': plant['plant_name'],
+        'plantType': plant['plant_type'],
+        'timestamp': plant['timestamp'],
+        'imageURL': plant['image_url'],
+        'care': plant['care']
+    }
+    return jsonify({'plant': response})
```

Comment:
We may end up scrapping this endpoint later, it depends on if the front-end thinks they will need to request a single plant ever